src/components/api.js

```diff
@@ -0,0 +1,223 @@
+export default class Api {
+  constructor(options) {}
+
+  //methods
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        authorization: "4792ec92-cf1c-45a6-8740-0f5d63585faa",
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          //check for successful response
+          console.log(res.ok);
+          return res.json();
+        }
+        //check for unsuccessful response
+        console.log("Error");
+        return Promise.reject(`getInitialCards method Error: ${res.status}`);
+      }) //process errors
+      .catch((err) => console.error(err));
+  }
+
+  getUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
+      headers: {
+        authorization: "4792ec92-cf1c-45a6-8740-0f5d63585faa",
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          /* console.log(res);  */ //console log response for debugging
+          return res.json(); //then return parsed response
+        }
+        //check for unsuccessful response
+        console.log("Error");
+        return Promise.reject(`getUserInfo method Error: ${res.status}`);
+      })
+      .catch((err) => console.error(err));
+  }
+
+  getUserAndCards() {
+    //called to get all info to display on page
+    return Promise.all([
+      this.getUserInfo(),
+      this.getInitialCards(),
+      this.updateProfilePic(),
```

Comment:
`updateProfilePic` is not needed here because it's the same request as `getUserInfo`